yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1776,32 +1777,34 @@ module vyatta-protocols-frr-bgp-v1 {
 						uses parameters-distance;
 						uses parameters-dampening;
 					}
-          container label {
-            description "label value for VRF";
-            configd:help "label value for VRF";
-            container vpn {
-              description "Between current address-family and vpn";
-              configd:help "Between current address-family and vpn";
-              leaf export {
-                description "For routes leaked from current address-family to vpn";
-                configd:help "For routes leaked from current address-family to vpn";
-                type uint32 {
-                  range 0..1048575;
-                }
-              }
-            }
-          }
+
+					container label {
+						description "label value for VRF";
+						configd:help "label value for VRF";
+						container vpn {
+							description "Between current address-family and vpn";
+							configd:help "Between current address-family and vpn";
+							leaf export {
+								description "For routes leaked from current address-family to vpn";
+								configd:help "For routes leaked from current address-family to vpn";
+								type uint32 {
+									range 0..1048575;
+								}
+							}
+						}
+					}
+
 					uses ipv4-aggregate-address;
 					uses network-ipv4;
 					uses redistribute-ipv4;
 				}
 
-        container ipv4-vpn {
-          description "BGP IPv4 VPN settings";
-          presence "Enables BGP IPv4 VPN settings";
-          configd:help "BGP IPv4 VPN settings";
-          uses network-ipv4-vpn;
-        }
+				container ipv4-vpn {
```

Comment:
Rather than use container ipv4-vpn, can this be changed to vpnv4-unicast ? I understand that the ipv4-vpn is used by the underlying frr configuration, but the vpnv4-unicast / vpnv6-unicast  naming is in line with other vendors usage. The mapping would then be from vpnv4-unicast to ipv4-vpn to generate the corresponding frr config from the yang.

Reply:
> Rather than use container ipv4-vpn, can this be changed to vpnv4-unicast ? I understand that the ipv4-vpn is used by the underlying frr configuration, but the vpnv4-unicast / vpnv6-unicast naming is in line with other vendors usage. The mapping would then be from vpnv4-unicast to ipv4-vpn to generate the corresponding frr config from the yang.

Hi , 

We took a note for the change you mentioned here. 
When we send PR on the commands for supporting 6PE and 6VPE  we will rectify that. 

Thanks, 
Kaushik

Reply:
looks good, Thank You,